src/common/storage/collection_and_field_names.py

```diff
@@ -102,3 +102,7 @@
 # Used for marking matches when returning data to the user and they select match marking vs.
 # filtering on a match.
 FLD_GENOME_ATTRIBS_MATCHED = "__match__"
+
+# Used for marking selections in the Arango collection; contains a list of internal selection IDs.
+# Underscore to separate from "real" attribs
+FLD_GENOME_ATTRIBS_SELECTIONS = "_selections"
```

Comment:
Why the different key format than `__match__`?

Reply:
would prefer the same for clarity unless I'm missing something

Reply:
This is an internal only key, it's not exposed by the API. The selection mark key will be `__sel__`